docs/source/en/_toctree.yml

```diff
@@ -1,144 +1,158 @@
-- sections:
+- title: Get started
+  sections:
   - local: index
-    title: 🤗 Transformers
+    title: Transformers
   - local: quicktour
-    title: Quick tour
+    title: Quicktour
   - local: installation
     title: Installation
-  title: Get started
-- sections:
-  - local: pipeline_tutorial
-    title: Run inference with pipelines
-  - local: autoclass_tutorial
-    title: Write portable code with AutoClass
-  - local: preprocessing
-    title: Preprocess data
-  - local: training
-    title: Fine-tune a pretrained model
-  - local: run_scripts
-    title: Train with a script
-  - local: accelerate
-    title: Set up distributed training with 🤗 Accelerate
-  - local: peft
-    title: Load and train adapters with 🤗 PEFT
-  - local: model_sharing
-    title: Share your model
-  - local: agents
-    title: Agents
-  - local: llm_tutorial
-    title: Generation with LLMs
-  - local: conversations
-    title: Chatting with Transformers
-  title: Tutorials
-- sections:
-  - isExpanded: false
+- title: Pretrained models
```

Comment:
Maybe we can come up with a better title here, as it's not only about models.

Reply:
Changed to "Base classes"